multichain-testing/test/chain-queries.test.ts

```diff
@@ -200,14 +202,16 @@ const queryChainWithoutICQ = test.macro({
     });
 
     const offerResult = await retryUntilCondition(
-      () => vstorageClient.queryData(`published.wallet.${agoricAddr}`),
+      () => smartWalletKit.readPublished(`wallet.${agoricAddr}`),
```

Comment:
wow there are a lot of these.

Reply:
yeah. I think `SmartWalletKit` is a good place to hang something like "await a matching state of the status node and return it"